packages/titus-backend-typescript/lib/plugins/auth/cognito/index.ts

```diff
@@ -0,0 +1,30 @@
+import fp from 'fastify-plugin'
+import buildGetJwks from 'get-jwks'
+
+function authenticate(request) {
+  return request.jwtVerify()
+}
+
+async function cognito(server, options) {
+  const getJwks = buildGetJwks()
+
+  server.register(require('fastify-jwt'), {
```

Comment:
NIT: For consistency, I think it's best to always use `import` vs mixing both `import` and `require`

Reply:
I totally agree. I haven't noticed this require. Will fix this

Reply:
pushed now